.github/workflows/long-running.yml

```diff
@@ -9,3 +9,9 @@ jobs:
     steps:
       - name: Sleep
         run: sleep 10s
+  
+  additional_job:
```

Comment:
This test file is to test long running jobs. What is the purpose of your test ? 
If it is for testing log retrieving, it should be in its own test file.